editor.planx.uk/src/@planx/components/Filter/model.ts

```diff
@@ -1,5 +0,0 @@
-export interface Filter {
-  id?: string;
-  handleSubmit?: (d: any, children: any) => void;
-  node?: any;
-}
```

Comment:
This was unused! Removed to avoid confusion, we're simply reading props defined at the top of Editor.tsx.